database/src/tests/mod.rs

```diff
@@ -16,18 +16,19 @@ enum TestDb {
 /// Represents a connection to a Postgres database that can be
 /// used in integration tests to test logic that interacts with
 /// a database.
-pub(crate) struct TestContext {
+pub struct TestContext {
     test_db: TestDb,
     // Pre-cached client to avoid creating unnecessary connections in tests
     client: Pool,
 }
 
 impl TestContext {
+    #[allow(dead_code)]
```

Comment:
Let's just `#![allow(dead_code)]` this whole module, will be easier I guess.

Reply:
:+1: 88a431cb8a591ebca195481a1985b58f01b2f628